StreamingAppDemo/app/src/main/java/com/fortyseven/degrees/streamingapp/home/HomeFragment.kt

```diff
@@ -66,26 +66,26 @@ class HomeFragment : Fragment() {
     ): Observable<Unit> =
         Observable.fromCallable {
             when (state) {
-                HomeVM.Empty -> Snackbar.make(
-                    view,
-                    "Design nice empty state",
-                    BaseTransientBottomBar.LENGTH_SHORT
-                ).show()
+                HomeVM.Idle -> {
+                    view.pullToRefresh.isRefreshing = false
+                }
                 HomeVM.Loading -> {
-                    Unit
+                    view.pullToRefresh.isRefreshing = true
```

Comment:
This is idempotent, will not make a difference if it's set to `true` multiple times (in case we switch to `Loading` twice in a row in our flow).